main/lsp/lsp_helpers.cc

```diff
@@ -227,6 +227,8 @@ bool hasSimilarName(const core::GlobalState &gs, core::NameRef name, string_view
     return fnd != string_view::npos;
 }
 
+const int multilineCutoff = 4;
```

Comment:
You can make this a `constexpr int MULTI_LINE_CUTOFF = 4;`, since we don't need a full const variable.

Then, comment it with something like:

// Max number of parameters in sig before printing as a multi-line sig.